src/App-basic-features.spec.ts

```diff
@@ -466,9 +466,10 @@ describe('App basic features', () => {
           withNoopAppMetadata(),
           withSuccessfulBotUserFetchingWebClient('B_FAKE_BOT_ID', 'U_FAKE_BOT_USER_ID'),
         );
+        const fakeLogger = createFakeLogger();
         const MockApp = await importApp(overrides);
         // Act
-        const app = new MockApp({ token: '', appToken: '', developerMode: true });
+        const app = new MockApp({ logger: fakeLogger, token: '', appToken: '', developerMode: true });
```

Comment:
Unrelated to the ExpressReceiver pull requests, but the extra output as I was iterating on tests was annoying me so I decided to silence them.